### ISSUE TITLE
Add a New Resource Page in the Admin Interface Lands Focus on the Footer When Navigating the Page With a Screen Reader

### ISSUE THREAD

**Issue by Santoz2** [ManifoldScholar/manifold#2435]
When attempting to add a new resource to a book, a user using a screen reader may be placed at the very end of the page, missing all of the page's content.

## Expected Behavior
When using a link to navigate to a different page, focus should be placed in an area that makes it easy for the reader to explore all of the content of the page. Often Manifold places focus on the first editable item; link behaviors should be consistent across the application.

## Current Behavior
When using the "Add a New Resource" link on the "Resources" page, the user may be brought to the bottom of the page when they attempt to navigate it.

Hadi believes that this link is acting somewhat like an anchor link, which typically is used to move the user around a single page), but it is worth noting that we are not sure why the link is behaving as it does.

## Steps to Reproduce
1. Navigate to Admin Interface (via Enter Admin Mode)
2. Navigate to an editable project
3. Navigate to the Resources page
4. Activate the "Add a New Resource" link
5. With a screen reader that has Browse Mode enabled (NVDA Modifier + Space if using NVDA), press the down arrow once.
6. Note that focus is placed in the Search Bar in the footer of the page.

## Testing Environment
NVDA 2019.2.1
ESR Firefox 68.2.0 esr

**Comment by dananjohnson** [comment]
@Santoz2 I'm unable to reproduce this issue testing in Firefox, Chrome, Safari, and using VoiceOver and NVDA screen readers. Clicking "Add a new resource" actually takes one to a unique route (`backend/projects/PROJECT_ID/resources/new`), so I'm not sure how focus could land in the footer. In all tests I did, focus landed in `<main>` when the new route loaded.

**Comment by dananjohnson** [comment]
Closing this issue until we can reliably reproduce it.

**Comment by Santoz2** [comment]
@dananjohnson I can reproduce this issue, and have made a screen recording of going through the issue.

This can be viewed via this dropbox link: 
[Screen Recording of the Add a New Resource Focus Issue](https://www.dropbox.com/s/fvkuulk5n25jcfe/AddANewResourceContentInfoIssue.mp4?dl=0)

For context, a screen reader user uses browse mode upon landing on a new page to discover the content on that page. If a user enabled browse mode on this page, and used the arrow down key to discover new content, they would immediately get lost by being jumped down to the bottom of the page.

However, this issue is pretty specific, and is not an issue that we would consider a deployment blocker because it is only a confusing setback, and does not prevent users from completing their intended action.

**Comment by dananjohnson** [comment]
@Santoz2 thanks so much for the detailed video! I'll re-open this issue and we'll do a deeper dive to hopefully uncover the cause. If you detect this happening anywhere else in the application, please make a note of it here. Thanks!

**Comment by zdavis** [comment]
Adjusted issue title as part of routine issue cleanup. Was "Add a New Resource Page in the Admin Interface Lands Focus on the Footer When Navigating the Page With a Screen Reader"

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by Santoz2** [comment]
Accessibility Triage Notes: This is considered a priority 3 issue, as it can be confusing, but  can be easily overcome by returning to the top of the page by way of either landmarks or headings.

**Comment by dananjohnson** [comment]
@Santoz2 I'm afraid that in the time since you first shared it, the link to the video above ([link](https://www.dropbox.com/s/fvkuulk5n25jcfe/AddANewResourceContentInfoIssue.mp4?dl=0)) no longer works. Could you please re-share it and confirm that the issue still exists for you? I'm still unable to reproduce it at this time. Thanks for your help!

**Comment by avo4-1925562** [comment]
@dananjohnson Zaijen is no longer on our team, but I'd be happy to re-share with you the screen recording demonstrating this issue: [Screen Recording of the Add a New Resource Focus Issue](https://www.dropbox.com/s/tmbf00ovypeb47t/AddANewResourceContentInfoIssue.mp4?dl=0). I have just retested the issue, and I can confirm that it still exists. I am using Firefox 92.0 (64-bit) and NVDA 2020.4. Please let me know if you have any other questions!

**Comment by dananjohnson** [comment]
Thank you for the video and for confirming the issue, @avo4-1925562! I will continue to investigate it!
